ui/src/app/views/signed-in/component.ts

```diff
@@ -51,18 +52,21 @@ export class SignedInComponent implements OnInit {
   ngOnInit(): void {
 
     this.profileImage = this.signInService.profileImage;
```

Comment:
Not really part of this PR, but why would this value be available before the user has signed in?

Reply:
Fair, will move.